eco.Rmd

```diff
@@ -5,8 +5,11 @@
 This chapter assumes you have a strong grasp of spatial data analysis and processing, covered in chapters 2-5.
 In it you will make use of R's interfaces to dedicated GIS software, and spatial cross validation, topics covered in chapters xx and xx respectively.
 
+## Introduction
+
 keywords: species distribution modeling, floristic modeling, spatial cross-validation, spatial autocorrelation, ordination (NMDS, Isomap), predictive mapping
 
+
 structure:
 1. intro ecological modeling and related applications beyond ecology
 2. introduce research question, shortly the dataset and the planned approach
```

Comment:
I suspect 1. and 2. could usefully be merged: ecological modeling is surely driven by research questions. I think a sentence like "Imagine you want to find out about the distribution of species x in ecosystem y. That is the basis of ecological modeling" to help demonstrate what you mean.

Reply:
this was just a structuring for me outlining what I would like to do. Of course, these points don't have to become subheadings.